app/pages/sections/supplier/supplier/ui-tests/general.ui.test.js

```diff
@@ -53,7 +53,7 @@ const pageSetup = async (setup = defaultPageSetup) => {
 
 const getLocation = ClientFunction(() => document.location.href);
 
-fixture('Supplier page - general')
+fixture.skip('Supplier page - general')
```

Comment:
Can you look at re-enabling this after merge?

Reply:
Sure, I'll put a task on the board